components/usage-api/usage/v1/usage.proto

```diff
@@ -106,12 +117,13 @@ message GetCostCenterResponse {
     CostCenter cost_center = 1;
 }
 
+enum BillingStrategy {
+    BILLING_STRATEGY_STRIPE = 0;
+    BILLING_STRATEGY_OTHER = 1;
+}
```

Comment:
Because all fields in a protobuf message are optional, and protobuf defaults to the empty value, for Enums it will default to `BILLING_STRATEGY_STRIPE` in this struct, when not set. That's not always desirable, as if you forget to set it it will have adverse effects. For that reason, the UNKNOWN value is normally used in any enum (the RPC is then implemented to fail if the value is UNKNOWN.

```suggestion
enum BillingStrategy {
    BILLING_STRATEGY_UNKNOWN = 0
    BILLING_STRATEGY_STRIPE = 1;
    BILLING_STRATEGY_OTHER = 2;
}
```